src/Usages/ClassConstantUsages.php

```diff
@@ -84,13 +84,16 @@ public function processNode(Node $node, Scope $scope): array
 			throw new ShouldNotHappenException(sprintf('$node->name should be %s but is %s', Identifier::class, get_class($node->name)));
 		}
 		$constant = (string)$node->name;
-		$usedOnType = $this->typeResolver->getType($node->class, $scope);
+
+		$type = $this->typeResolver->getType($node->class, $scope);
```

Comment:
ℹ️ Need to keep `$type` as separate variable for error reporting purposes, otherwise error format for `constantUsages.php` will change from `Cannot access constant GLITTER on X` to `Cannot access constant GLITTER on object`